plugins/lookup/hashi_vault.py

```diff
@@ -158,6 +158,17 @@
         - name: EC2_REGION
         - name: AWS_REGION
         version_added: '0.2.0'
+    action_type:
+      description:
+        - The action type to be performed for the provided path
+      ini:
+        - section: lookup_hashi_vault
+          key: action_type
+          version_added: '0.2.0'
+      choices:
+        - READ
```

Comment:
Is there a reason you are using upper case for the choice names?

Reply:
My analysis started with `curl` where the HTTP method used to browse Vault is `LIST`
I initially listed choices such as `GET` and `LIST`, but it seemed more reasonable to preserve the HVAC API method names - read & list.
At that point however, I did not consider letter cases. 

Since the preferred is lower case, and we are not naming HTTP Methods, I will refactor these to lower case :)

Reply:
https://github.com/ansible-collections/community.general/pull/664/commits/3d624beaf42e740a8f26de3bd5001f1e8d11e503